Examples/Python/python/acts/examples/itk.py

```diff
@@ -71,8 +71,8 @@ def buildITkGeometry(
                 binToleranceZ=(5 * u.mm, 5 * u.mm),
                 binTolerancePhi=(0.025 * u.mm, 0.025 * u.mm),
                 layers=LayerTriplet(True),
-                subVolumeName=LayerTriplet("Pixel::Pixel"),
-                sensitiveNames=LayerTriplet(["Pixel::siLog"]),
+                subVolumeName=LayerTriplet("ITkPixel__ITkPixelDetector"),
```

Comment:
There is realy nothing to comment on here. The names are as they are. 
But I have two suggestions for this file style.
1) maybe a comment (next to this place could be given how this names can be found actually? I.e. It is not any name, and was not guessed. 

2) This file has several very long functions. Can this be partitioned in a more human/screen size chunks? Looking at this very place it seems we could have function: getInnerPixelsVolume, getOuterPixelsVolume, .... 

Maybe this is for another PR.

Reply:
Both good suggestions, but may be tricky to do.
1. Getting the names was not easy, and I still might not have done it quite right - except that I get very similar performance at the end. I used a combination of adding `printf`s in the loading code (useful to find what it found before), printing object descendants from root (good for making long lists, but not everything easy to get at), and browsing with the `TGeoManager` with the new JSROOT TBrowser (easy to navigate, but have to scan through long lists by eye). Hopefully we won't have another major renaming, so maybe this is the last time.
2. This case replaces the even more verbose JSON 😄 Thinking about this, I don't know if it is clearer having it altogether, or split up.
Can we save this for another PR? It would be good to get this in before we have people hitting an incompatibility with `acts-itk` repo. We'll probably get clashes when people update one or the other, but then we'll be able to suggest to update to the latest for both.

Reply:
ad 2). Sure. It makes sense to leave it. 
ad 1) "we won't have another major renaming" - I would not count on that, the ITk will be around for a while. 
Some info - along what you just wrote should be somehow available in the code imho. Maybe in the preamble in this file?